Tests/UnitTests/Core/Surfaces/DiscSurfaceTests.cpp

```diff
@@ -37,16 +37,13 @@ BOOST_AUTO_TEST_CASE(DiscSurfaceConstruction) {
   // scaffolding...
   double rMin(1.0), rMax(5.0), halfPhiSector(M_PI / 8.);
   //
-  /// Test DiscSurface fully specified constructor but no transform
-  BOOST_CHECK_NO_THROW(
-      Surface::makeShared<DiscSurface>(nullptr, rMin, rMax, halfPhiSector));
```

Comment:
Why is this test removed?

Reply:
This was a special test for DiscSurface w/o transform, that doesn't exist anymore as no transform is not an option.

Reply:
Alright